controllers/vspherecluster_controller.go

```diff
@@ -85,6 +86,8 @@ func AddClusterControllerToManager(ctx *context.ControllerManagerContext, mgr ma
 	return ctrl.NewControllerManagedBy(mgr).
 		// Watch the controlled, infrastructure resource.
 		For(clusterControlledType).
+		// Filter events by watch filter and paused labels.
+		WithEventFilter(ctx.GetCommonEventFilter()).
```

Comment:
Unsure why this controller doesn't have the secondary `Cluster` watch like the others do.